docs/processing_information.md

```diff
@@ -64,19 +64,33 @@ Prior to normalization, low-quality cells are removed from the gene by cell coun
 To identify low-quality cells, we use [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a package that jointly models proportion of reads belonging to mitochondrial genes and number of unique genes detected.
 Cells with a high likelihood of being compromised (greater than 0.75) and cells that do not pass a minimum number of unique genes detected threshold of 200 are removed from the counts matrix present in the `_processed.rds` file.
 
+If CITE-seq data is present, additional low-quality cells are further removed based on antibody-derived tag (ADT) content, as described in [Processed CITE-seq data](#processed-cite-seq-data).
+
 Log-normalized counts are calculated using the deconvolution method presented in [Lun, Bach, and Marioni (2016)](https://doi.org/10.1186/s13059-016-0947-7).
 The log-normalized counts are used to model variance of each gene prior to selecting the top 2000 highly variable genes (HVGs).
 These HVGs are then used as input to principal component analysis, and the top 50 principal components are selected.
 Finally, these principal components are used to calculate the [UMAP (Uniform Manifold Approximation and Projection)](http://bioconductor.org/books/3.13/OSCA.basic/dimensionality-reduction.html#uniform-manifold-approximation-and-projection) embeddings.
 
+### Processed CITE-seq data
```

Comment:
I would move this section to a subsection of `CITE-seq quantification`, as it seems odd to talk about the filtering before quantification.

Reply:
Done in https://github.com/AlexsLemonade/scpca-docs/pull/110/commits/1e3f1c4b35474acbca477d047671462dcc52d848 (w/ cleanup in https://github.com/AlexsLemonade/scpca-docs/pull/110/commits/c77beae7761604bc700c1e47a5ce241374595822 ).